ChildProject/tables.py

```diff
@@ -13,7 +13,13 @@ def __init__(self, name: str, missing: Set):
         super().__init__(
             f"dataframe {name} misses the following required columns: {missing}"
         )
+        
+class IncorrectDtypeException(Exception):
+    def __init__(self, error: str):
 
+        super().__init__(
+            error
+        )
```

Comment:
This is a tiny bit confusing to me. Was `__init__` overridden here, so that `IncorrectDtypeException` will only accept a single argument?

Reply:
Also, `error` is an unusual (to me) name for the error message. I've mostly seen `message` or `msg`.

Reply:
You are right, it does not need to be overridden, the exception was originally taking multiple arguments and was in the end simplified. So removing the __init__ makes sense. (and killing two birds with one stone, this will remove the unsual ` error` argument).